scrapinghub/client.py

```diff
@@ -103,24 +213,78 @@ def __init__(self, client, projectid):
 
 
 class Spiders(object):
+    """Class to work with collection of project spiders.
+
+    Attributes:
+        projectid (int): project id
+
+    Use Project instance to get a Spiders instance.
+
+    Example:
+        >>> project.spiders
+        <scrapinghub.client.Spiders at 0x1049ca630>
+    """
 
     def __init__(self, client, projectid):
         self.projectid = projectid
         self._client = client
 
     def get(self, spidername, **params):
+        """Get a Spider instance for a given spider name.
+
+        The method does a request to get spider id (and check if it exists).
+
+        Args:
+            spidername (str): a spider name
```

Comment:
I expect it to be formatted using sphinx and rst syntax, like https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/engine/base.py#L23 or https://github.com/kennethreitz/requests/blob/master/requests/models.py#L194

in that case it'll be easy to provide some nice RTD documentation for the client